caab-service/src/main/java/uk/gov/laa/ccms/caab/api/entity/Application.java

```diff
@@ -1,17 +1,18 @@
 package uk.gov.laa.ccms.caab.api.entity;
 
 
-
 import jakarta.persistence.CascadeType;
 import jakarta.persistence.Column;
 import jakarta.persistence.Convert;
 import jakarta.persistence.Embedded;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
 import jakarta.persistence.FetchType;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.Id;
 import jakarta.persistence.JoinColumn;
 import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MapKey;
```

Comment:
Does this class need the imports updating? Not sure this and the hibernate annotations are used.

Reply:
imports removed